src/ConnectedField.js

```diff
@@ -72,7 +71,10 @@ const createConnectedField = ({
         _value: ownProps.value // save value passed in (for checkboxes)
       }
     },
-    actions,
+    (dispatch, initialProps) => mapValues({ blur, change, focus }, actionCreator => (...args) => {
```

Comment:
This is really suboptimal. Action creators will be rebound on every render. Maybe we could assert that `dispatch` would never change and ACs once:

```js
(initialDispatch, initialProps) => {
  const dispatch = initialProps.dispatch || initialDispatch
  const boundActions = boundActionCreators(actions, dispatch)

  return () => boundActions
}
```